showcase/app/controllers/page-components.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { scheduleOnce } from '@ember/runloop';
+
+import type RouterService from '@ember/routing/router-service';
+import type Owner from '@ember/owner';
+
+export default class PageComponentsController extends Controller {
+  @service declare router: RouterService;
```

Comment:
```suggestion
  @service declare readonly router: RouterService;
```
[Suggestion] In the converted utilities file we add `readonly` to the `router`. We could do the same here.